Omulosi/iReporter
# #162365733 Fix clear_all method in database model and remove redundant data

#### What does this PR do?
Fixes `clear_all` method in database model and removes redundant data
#### Description of Task to be completed?
Ensure the clear_all method of the record model clears all data in the internal database and returns None
#### How should this be manually tested?
clone the repo , navigate to the bg-fix-db-model-162365733 branch and run unit tests. The unit tests covering the fix should pass
#### What are the relevant pivotal tracker stories?
#162365733
